media-libs/opencv/opencv-3.2.0-r1.ebuild

```diff
@@ -233,20 +269,19 @@ src_configure() {
 		-DENABLE_SOLUTION_FOLDERS=OFF
 		-DENABLE_PROFILING=OFF
 		-DENABLE_COVERAGE=OFF
-		-DENABLE_OMIT_FRAME_POINTER=OFF
+		-DENABLE_OMIT_FRAME_POINTER=${ENABLE_OMIT_FRAME_POINTER}
 		-DENABLE_FAST_MATH=OFF
-		-DENABLE_SSE=OFF
-		-DENABLE_SSE2=OFF
-		-DENABLE_SSE3=OFF
-		-DENABLE_SSSE3=OFF
-		-DENABLE_SSE41=OFF
-		-DENABLE_SSE42=OFF
-		-DENABLE_POPCNT=OFF
-		-DENABLE_AVX=OFF
-		-DENABLE_AVX=OFF
-		-DENABLE_AVX2=OFF
-		-DENABLE_FMA3=OFF
-		-DENABLE_NEON=OFF
+		-DENABLE_SSE=$(usex cpu_flags_x86_sse)
```

Comment:
Using those flags to control -m options is wrong. They are only for assembly and similar cases.